packages/server/server/api/trecipe/trecipe.controller.ts

```diff
@@ -60,17 +70,25 @@ class TrecipeController implements Controller {
 
     private deleteTrecipeById(req: Request, res: Response, next: NextFunction) {
         const uuid: string = req.params.id;
-        TrecipeService.deleteTrecipeById(uuid)
+        const user = req.user as User;
+        TrecipeService.deleteTrecipeById(uuid, user)
             .then((deletedCount) => {
-                res.status(200).json({ deletedCount: deletedCount });
+                UserService.updateUserByUsername(user.username, {
+                    trecipes: user.trecipes.filter((trecipeId) => {
+                        return trecipeId !== uuid;
+                    }),
+                }).then(() => {
```

Comment:
Let's chain this `then` instead of nesting.

Reply:
Okie, will do thanks! 👍